README.md

```diff
@@ -1,8 +1,8 @@
 # QCCalculator
 
-Development for python driven QC calculation (QCCalculator) and presentation (QSpector)
+Development for python driven QC calculation (QCCalculator)
 
-## Install 
+## Install
 pip install -U git+https://github.com/bigbio/qccalculator.git#egg=QCCalculator
```

Comment:
I think the egg name needs to be same as package name -> lower case

Reply:
Done